BlogTemplate/Pages/Edit.cshtml

```diff
@@ -1,6 +1,6 @@
 @page "{slug}"
-@using BlogTemplate
-@using BlogTemplate.Models

```

Comment:
can you reorganize these so the `@using` statements are all together and sorted?